app/views/calculator/payment_summary.scala.html

```diff
@@ -37,10 +37,8 @@
 
 @main(
     title = Messages("ssttp.calculator.form.payment_summary.title"),
-    loggedIn = true
-) {
-    <a class="link-back" href="@ssttpcalculator.routes.CalculatorController.getPayTodayQuestion()">@Messages("ssttp.calculator.form.back")</a>
-
+    loggedIn = true,
```

Comment:
Should this one be getPayTodayQuestion ?

Reply:
Good question - the answer is yes it should but the ticket wasn't clear enough.